docs/guides/jupyter.rst

```diff
@@ -0,0 +1,363 @@
+  and features `several examples and how-to guides using
+  notebooks <https://docs.poliastro.space/en/stable/gallery.html>`_
+  and displays them in an appealing thumbnail gallery.
+  In addition, `poliastro uses unpaired MyST
+  Notebooks <https://github.com/poliastro/poliastro/tree/0.15.x/docs/source/examples>`_
+  to reduce repository size and improve integration with git.
+
+Background on alternative notebook formats
+------------------------------------------
+
+Jupyter notebooks in ``.ipynb`` format
+(as described in the `nbformat
+documentation <https://nbformat.readthedocs.io/>`_)
+are by far the most widely used for historical reasons.
+
+However, to compensate some of the disadvantages of the ``.ipynb`` format
+(like cumbersome integration with version control systems),
+`jupytext`_ offers :doc:`other formats <jupytext:formats>`
+based on plain text rather than JSON.
+
+As a result, there are three modes of operation:
+
+- Using classic ``.ipynb`` notebooks. It's the most straightforward option,
+  since all the tooling is prepared to work with them,
+  and does not require additional pieces of software.
+  It is therefore simpler to manage, since there are fewer moving parts.
+  However, it requires some care when working with Version Control Systems (like git),
+  by doing one of these things:
+
+  - Clear outputs before commit.
+    Minimizes conflicts, but might defeat the purpose of notebooks themselves,
+    since the computation results are not stored.
+  - Use tools like `nbdime <https://nbdime.readthedocs.io/>`_ (open source)
+    or `ReviewNB <https://www.reviewnb.com/>`_ (proprietary)
+    to improve the review process.
+  - Use a different collaboration workflow that doesn't involve notebooks.
+
+- Replace ``.ipynb`` notebooks with :doc:`a text-based format <jupytext:formats>`.
+  These formats behave better under version control
+  and they can also be edited with normal text editors
+  that do not support cell-based JSON notebooks.
+  However, text-based formats do not store the outputs of the cells,
+  and this might not be what you want.
+- Pairing ``.ipynb`` notebooks with a text-based format,
+  and putting the text-based file in version control,
+  as suggested in the :doc:`jupytext documentation <jupytext:paired-notebooks>`.
+  This solution has the best of both worlds.
+  In some rare cases you might experience synchronization issues between both files.
+
+These approaches are not mutually exclusive,
+nor you have to use a single format for all your notebooks.
+For the examples in this document, we will use the :ref:`MyST Markdown
+format <jupytext:formats.md#myst-markdown>`.
+
+If you are using alternative formats for Jupyter notebooks,
+you can include them in your Sphinx documentation
+using either `nbsphinx`_ or `MyST-NB`_
+(:ref:`see above <nb-background>`
+for more information about the differences between them).
+
+.. _other-formats:
+
+Using notebooks in other formats
```

Comment:
I'd probably move this higher up because it's the information most users will be interested in, and move the "background explanation" stuff about notebook formats beneath it with a cross-link.

Reply:
Good point, I will move the "background" to the end of the document (and change "above" by "below" etc)